.github/workflows/pr_push.yml

```diff
@@ -94,7 +94,7 @@ jobs:
     - name: Install apt packages
       run: |
         sudo apt-get update
-        sudo apt-get install -y cmake clang-format-15 cmake-format libhwloc-dev
+        sudo apt-get install -y black cmake clang-format-15 cmake-format libhwloc-dev
```

Comment:
Can you use a pip package instead? I'm not sure if find_package() will work for pip modules, but this way it will be more convenient to use `black` on Windows.

Reply:
hmm I'm not sure what do you mean... In this job, as you can see, we use `apt` for all tools. This is because this job is only run on Linux.

Anyway, black is added as a python dep. in our `requirements.txt` file, so it should work well on all OSes. I've verified it on my machine with pip installation.

Please let me know if you propose to change something

Reply:
@PatKamin This job (`Coding style`) is run only on Linux Ubuntu